src/devices/fineoffset_ws90.c

```diff
@@ -109,9 +109,6 @@ static int fineoffset_ws90_decode(r_device *decoder, bitbuffer_t *bitbuffer)
     int supercap_V  = (b[21] & 0x3f);
     int firmware    = b[29];
 
-    if (battery_lvl > 100) // More then 100%?
```

Comment:
The comment about 1.5 vs 1.4 and the code do not line up at all.  The code is mapping 1.4 to 3.0 to be 0 to 100.  That doesn't make a lot of sense.   Also "lithium battery" is confusing, as there is lithium primary, lithion ion, and lithium iron phosphate.   I'm really not sure which kind we are talking about.

This comment may amount to "the code as it exists needs more help than your PR gives it" which might not be fair.